context-monitoring/src/main/java/de/atb/context/monitoring/monitors/file/FileSystemMonitor.java

```diff
@@ -158,174 +71,21 @@ protected final void iterateFiles(final File directory) {
             try {
```

Comment:
`files` should be checked for `null` before the `for` loop.

Reply:
[x] done